apps/feature/behaviours/get-form-session.js

```diff
@@ -44,7 +44,7 @@ module.exports = superclass => class extends superclass {
               const updated = moment(form.updated_at);
               const formSession = {
                 id: form.id,
-                reference: JSON.stringify(form.session.reference),
+                reference: (form.session.reference),
```

Comment:
Can remove the brackets if you are not using the stringily method.

Reply:
removed the brackets